### DISCUSSION TITLE
为什么yolov8结构图中第13层上采样（Unsample）后通道数会减半

### DISCUSSION THREAD

**Discussion by zyw-stu** [open-mmlab/mmyolo#620] [Q&A] [answered]
我想请问一下yolov8结构图的第13层，为什么40×40×512×w在上采样upsample之后尺寸变成了80×80×256×w （即通道数缩小为一半）。有点困惑，因为我记得nn.Unsample函数只改变的是h和w

**Comment by hhaAndroid** [comment]
@zyw-stu 不好意思，绘制错了，马上修复

**Reply by zyw-stu** [reply]
好的，谢谢您！

**Reply by 140ai** [reply]
@hhaAndroid 请问用什么软件绘制的网络结构图

**Comment by RangeKing** [comment, marked as answer]
@zyw-stu 感谢反馈，已在 PR https://github.com/open-mmlab/mmyolo/pull/621 中修正。

**Reply by zyw-stu** [reply]
看到啦，谢谢您！
